website/source/api/auth/gcp/index.html.md

```diff
@@ -0,0 +1,362 @@
+---
+layout: "api"
+page_title: "Google Cloud Platform Auth Backend - HTTP API"
+sidebar_current: "docs-http-auth-gcp"
```

Comment:
For this page to actually show up in the page, this sidebar link should be added to `source/layouts/api.erb` as follows:
Under `docs-http-auth` section
```
<li<%= sidebar_current("docs-http-auth-gcp") %>>
            <a href="/api/auth/gcp/index.html">GCP</a>
</li>
```